src/Symfony/Component/Notifier/Bridge/OneSignal/OneSignalOptions.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\OneSignal;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+use Symfony\Component\Notifier\Notification\Notification;
+
+/**
+ * @author Tomas Norkūnas <norkunas.tom@gmail.com>
+ */
+final class OneSignalOptions implements MessageOptionsInterface
+{
+    private $options;
+
+    public function __construct(array $options = [])
+    {
+        $this->options = $options;
+    }
+
+    public static function fromNotification(Notification $notification): self
+    {
+        $options = new self();
+        $options->headings(['en' => $notification->getSubject()]);
+        $options->contents(['en' => $notification->getContent()]);
+
+        return $options;
+    }
+
+    public function headings(array $headings): self
+    {
+        $this->options['headings'] = $headings;
+
+        return $this;
+    }
+
+    public function contents(array $contents): self
+    {
+        $this->options['contents'] = $contents;
+
+        return $this;
+    }
+
+    public function url(string $url): self
+    {
+        $this->options['url'] = $url;
+
+        return $this;
+    }
+
+    public function data(array $data): self
+    {
+        $this->options['data'] = $data;
+
+        return $this;
+    }
+
+    public function sendAfter(\DateTimeInterface $datetime): self
+    {
+        $this->options['send_after'] = $datetime->format('Y-m-d H:i:sO');
```

Comment:
Can you double check if this is the correct format.

Reply:
Yes it's correct